wandb/sdk/launch/deploys/systemd/launch_agent_service_setup.py

```diff
@@ -0,0 +1,54 @@
+import os
+import subprocess
+
+
+def check_pip():
+    try:
+        import pip
+    except ImportError:
+        print("pip not installed")
+        return False
+    return True
+
+
+def maybe_install_pip():
+    if not check_pip():
+        print("installing pip")
+
+        subprocess.check_call(["sudo", "yum", "install", "python3-pip"])
+
+
+def check_wandb():
+    try:
+        import wandb
+    except ImportError:
+        print("wandb not installed")
+        return False
+    return True
+
+
+def check_curl():
+    try:
+        subprocess.check_call(["which", "curl"])
+    except subprocess.CalledProcessError:
+        print("curl not installed")
+        return False
+
+
+def maybe_install_curl():
+    if not check_curl():
+        print("installing curl")
+        subprocess.check_call(["sudo", "yum", "install", "curl"])
```

Comment:
Assuming we have yum is probably bad right?  Also, what do we need curl for?

Reply:
Ah, this was just an initial attempt at setting it up on EC2. Removing the set up script and will have a documentation page that can handle general cases.

If this feature gains traction we'll set up a deb package for the launch agent